apps/coscrad-frontend/src/utils/generic-components/presenters/get-text-items-for-multilingual-text-presenter.ts

```diff
@@ -0,0 +1,37 @@
+import { IMultilingualTextItem } from '@coscrad/api-interfaces';
+import { isNullOrUndefined } from '@coscrad/validation-constraints';
+import { isInLanguage } from './is-in-language';
+import { isOriginalTextItem } from './is-original-text-item';
+
+interface TextItemsForMultilingualTextPresenterProps {
+    primaryMultilingualTextItem: IMultilingualTextItem;
+    translations: IMultilingualTextItem[];
+    isTranslated: boolean;
+}
+
+export const getTextItemsForMultilingualTextPresenter = (
```

Comment:
This is what I was looking for. I'd prefer to name it differently. Let's brainstorm on this with fresh heads before this branch gets merged.

Reply:
Also, if we can find a way to share the `MultinglinualText` class with the client, there's no need for any multilingual text utils on the front-end.